website/docs/packages/reference/query-module.md

```diff
@@ -24,7 +35,7 @@ The input property enables you to pass dynamic values from your app to the query
 You cannot edit the input name or query configuration from the App; you can only pass values to the available inputs. 
```

Comment:
Suggestion: what is we complement this session with information that it's possible to change the values of inputs in runtime by passing different values in the .run() function? I've attached a video example that shows that it's possible and the value passed in runtime has precedence over the default value set in the module instance page.

[Video example](https://drive.google.com/file/d/14Nk5fuIhR_l_hNfupUjAhHDzUpPRHag9/view?usp=sharing)